<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaRegistro</class>
 <widget class="QMainWindow" name="VistaRegistro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenULE - Registro Admin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <spacer name="verticalSpacerTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutFrame">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>

         <item><widget class="QLineEdit" name="lineEdit_nombre"><property name="placeholderText"><string>Nombre</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_apellido"><property name="placeholderText"><string>Apellido</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_correo"><property name="placeholderText"><string>Correo</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_password"><property name="echoMode"><enum>QLineEdit::Password</enum></property><property name="placeholderText"><string>Contraseña</string></property></widget></item>
         <item><widget class="QComboBox" name="comboBox_rol">
           <item><property name="text"><string>estudiante</string></property></item>
           <item><property name="text"><string>profesor</string></property></item>
           <item><property name="text"><string>personal_comedor</string></property></item>
           <item><property name="text"><string>administrador</string></property></item>
         </widget></item>
         <item><widget class="QLineEdit" name="lineEdit_tui"><property name="placeholderText"><string>TUI (solo estudiantes)</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_grado"><property name="placeholderText"><string>Grado académico</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_especialidad"><property name="placeholderText"><string>Especialidad (comedor)</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_dni"><property name="placeholderText"><string>DNI</string></property></widget></item>
         <item><widget class="QLineEdit" name="lineEdit_telefono"><property name="placeholderText"><string>Teléfono</string></property></widget></item>
         <item><widget class="QPushButton" name="pushButton_registrar"><property name="text"><string>Registrar</string></property></widget></item>
         <item><widget class="QPushButton" name="pushButton_volver"><property name="text"><string>Volver</string></property></widget></item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacerBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                        stop:0 #2e7d32, stop:1 #a5d6a7);
}
QLineEdit, QComboBox {
    border: 2px solid #ccc;
    border-radius: 10px;
    padding: 5px;
    background-color: rgba(255, 255, 255, 0.1);
    color: white;
}
QPushButton {
    border-radius: 10px;
    font-weight: bold;
    padding: 5px;
}
QPushButton#pushButton_registrar {
    background-color: #337ab7;
    color: white;
}
QPushButton#pushButton_volver {
    background-color: #e57373;
    color: white;
}
   </string>
  </property>
 </widget>
 <resources/>
 <connections/>
</ui>